### PR TITLE
127 I18N: translate searchUserAside

### PR DESCRIPTION
**Pull request by PCapid3v**

### CODE REVIEW

File: src/components/searchUserAside/index.js [hunk 1]
```diff
@@ -8,14 +8,18 @@ import UsersList from "../usersList"
 import UserSearchBar from "../userSearchBar/UserSearchBar"
 
 import { getUserByName } from "../../service/apiClient"
+import { useTranslation } from "react-i18next"
 
 import "./style.css"
+import { t } from "i18next"
```
Comment (PeachyOmnivore) [inline src/components/searchUserAside/index.js:14]: Don't think you need this as you have defined it already down below.

**Review by PeachyOmnivore — CHANGES_REQUESTED** [review]

File: src/components/searchUserAside/index.js (hunk 1 above)
Comment (PCapid3v) [reply src/components/searchUserAside/index.js:14]: Oh yeah! I should have use useTranslation instead 🤦‍♂️

**Review by Uprising5034 — COMMENTED** [review]
Looks good! Just Lukas' comment to address.

File: src/i18n.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ i18n
   .use(initReactI18next) // passes i18n down to react-i18next
   .init({
     resources,
-    fallbackLng: "en",
+    fallbackLng: "se",
```
Comment (PeachyOmnivore) [inline src/i18n.js:20]: Make sure not to leave this set as Swedish.

**Review by PeachyOmnivore — CHANGES_REQUESTED** [review]

**Review by PeachyOmnivore — COMMENTED** [review]
LGTM

**Review by Auenc — APPROVED** [review]

**Pull request merged**